packages/git/git.1.5.3/opam

```diff
@@ -57,7 +57,7 @@ depopts: [
 conflicts: [
  "cohttp"  {< "0.18.0"}
  "cohttp"  {>= "0.19.0"}
- "conduit" {< "0.8.4"}
+ "conduit" {< "0.8.4" | >= "3.0.0"}
```

Comment:
```suggestion
  "conduit" {< "0.8.4" }
  "conduit" {>= "3.0.0"}
```

Reply:
There are a lot of these for the various git versions. I stop marking them, we can fix them using a sed call all at once